### ISSUE TITLE
tctl should support creating client certificates

### ISSUE THREAD

**Issue by tooboredtocode** [gravitational/teleport#6847]
### What
The way teleport database access is currently set up, only the auth node (i think) can create client certificates, making it impossible to get certificates for applications relying on the database.
The tctl command could be expanded for such functionality and be able to create secure client certificates for applications.

### Workaround
Using the command used to sign server keys and certificates seems to work for now.
```
tctl auth sign --format=db --host=username --out=filename --ttl=8760h
```

**Comment by webvictim** [comment]
@r0mant I think this should be supported? Is it related to remote `tctl` usage and impersonation?

**Comment by r0mant** [comment]
@webvictim @tooboredtocode Yes, I merged a patch to make this work yesterday: https://github.com/gravitational/teleport/pull/7024. Should make it into 6.2.1.

To do this, the user should have permissions to impersonate system role `Db` representing a database node:

```yaml
impersonate:
  roles:
  - Db
  users:
  - Db
```

I'll add a discussions article in the meanwhile and then convert to the proper docs.

**Comment by Valien** [comment]
@r0mant answer solved my issues with Teleport cloud. It is case-sensitive as well `Db` vs `db`.

Might want to put in a Cloud-related line here -- https://goteleport.com/docs/ver/8.0/database-access/guides/mongodb-atlas/#step-23-configure-atlas - as the current command assumes users are probably self-hosted in which it will work with no issue.
